pkgs/development/python-modules/myst-parser/default.nix

```
@@ -0,0 +1,64 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flit-core
+, docutils
+, mdit-py-plugins
+, pyyaml
+, sphinx
+, typing-extensions
+  # , beautifulsoup4
+  # , coverage
+  # , pytest-cov
+  # , pytest-param-files
+  # , pytest-regressions
+  # , sphinx-pytest
+  # , pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "myst-parser";
+  version = "0.18.0";
+
```

The modules doesn't support [obsolete Python releases](https://github.com/executablebooks/MyST-Parser/blob/v0.18.0/pyproject.toml#L35).

```suggestion

  disabled = pythonOlder "3.7";

```